master/pkg/syncx/mapx/mapx_test.go

```diff
@@ -28,6 +30,40 @@ func FuzzMap(f *testing.F) {
 			m.WithLock(func(m map[uint8]string) {
 				_ = maps.Keys(m)
 			})
+		case 5:
+			_ = m.Values()
+		case 6:
+			m.Clear()
 		}
 	})
 }
+
+func TestMapx(t *testing.T) {
+	tests := []struct {
+		k string
+		v string
+	}{
+		{"1234", "hi"},
+		{"1235", "hello"},
+		{"1236", "world"},
+	}
+	testMap := New[string, string]()
+	for _, tt := range tests {
+		t.Run(tt.k, func(t *testing.T) {
```

Comment:
This is a strange use of `t.Run`. I would just use a for loop to insert the keys since each insertion isn't really a subtest

Reply:
Yeah, that somehow skipped through. updated as suggested.